install.md

```diff
@@ -70,6 +71,7 @@ and requires that you have permissions to administer your computer).
 sudo apt-get install -y software-properties-common
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 sudo apt update
+sudo apt-get upgrade
```

Comment:
im not sure if this is correct here since it just calls apt update and not apt-get update before

Reply:
It should be apt-get there, too (well, it works either way, but....)